client/my-sites/plugins/no-permissions-error.jsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Main from 'components/main';
+import EmptyContent from 'components/empty-content';
+import DocumentHead from 'components/data/document-head';
+import SidebarNavigation from 'my-sites/sidebar-navigation';
+
+function NoPermissionsError( { translate } ) {
```

Comment:
While this syntax is acceptable, we tend to use `const` in that case:

```
const NoPermissionsError = ( { translate } ) => {
...
};
```

also, since the component returns a single node and has no logic, we can simplify:

```
const NoPermissionsError = ( { translate } ) => {
	return (
		<Main>
			...
		</Main>
	);
};
```

to

```
const NoPermissionsError = ( { translate } ) => (
	<Main>
		...
	</Main>
);
```